src/Record.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+class Record {
+
+    private string $album_name;
+    private string $artist_name;
+    private int $year;
+    private string $record_label;
+    private int $number_of_tracks;
+
+    public function __construct(string $album_name, string $artist_name, int $year, string $record_label, int $number_of_tracks) 
+    {
+        $this->album_name = $album_name;
+        $this->artist_name = $artist_name;
+        $this->year = $year;
+        $this->record_label = $record_label;
+        $this->number_of_tracks = $number_of_tracks;
+    }
+
+    public function create_record_card(): string 
```

Comment:
as per PSR-1 coding standard for php, methods should be named using camelCase
<img width="755" alt="image" src="https://github.com/MrTKetchup/collection_app/assets/25844355/905d159e-10e7-49b9-b5fd-938a79be63ab">